### PR TITLE
Temporarily disable single-op tombstone

### PR DESCRIPTION
**Pull request by adamcfraser**
Need to disable single-op tombstone until they are properly indexed by the view engine. (https://issues.couchbase.com/browse/MB-25422)

### CODE REVIEW

File: base/bucket_gocb.go [hunk 1]
```diff
@@ -1413,10 +1413,34 @@ func (bucket CouchbaseBucketGoCB) WriteUpdateWithXattr(k string, xattrKey string
...
+				continue
+			} else {
+				// Non-recoverable error - return
+				return emptyCas, removeErr
+			}
+
+			// update xattr only
+			casOut, writeErr := bucket.WriteCasWithXattr(k, xattrKey, exp, cas, nil, updatedXattrValue)
+			if writeErr != nil && writeErr != gocb.ErrKeyExists && !isRecoverableGoCBError(writeErr) {
+				LogTo("CRUD", "Update of new value during WriteUpdateWithXattr failed for key %s: %v", k, writeErr)
```
Comment (tleyden) [inline base/bucket_gocb.go:1433]: Should this LogTo be a Warn instead?  Also, can you add something which explains the implications?  Like "The tombstone delete revision has not been recorded..."

File: base/bucket_gocb.go [hunk 2]
```diff
@@ -1413,10 +1413,34 @@ func (bucket CouchbaseBucketGoCB) WriteUpdateWithXattr(k string, xattrKey string
...
+				return emptyCas, writeErr
 			}
+
+			// If there was no error, we're done
+			if writeErr == nil {
```
Comment (tleyden) [inline base/bucket_gocb.go:1438]: Might be clearer if this code block was moved above the other `if writeErr != nil && writeErr != gocb.ErrKeyExists && !isRecoverableGoCBError(writeErr) ` block

File: rest/import_test.go [hunk 3]
```diff
@@ -334,3 +335,73 @@ func TestXattrDoubleDelete(t *testing.T) {
 	revId = body["rev"].(string)
 
 }
+
+func TestViewTombstoneRetrieval(t *testing.T) {
```
Comment (tleyden) [inline rest/import_test.go]: Can you rename this to TestViewQueryTombstoneRetrieval?  As it stands it sounds like it's just viewing (reading) a tombstone.  Adding "ViewQuery" might call out that it's a "CouchbaseView"

**Review by tleyden — COMMENTED** [review]
Do you know the sync gateway ticket where this was changed into a single op?  If so, I think it would be useful to mention that somewhere here so they get xlinked.

**Comment by adamcfraser** [comment]
Temporarily reverts tombstone single-op handling that was added in https://github.com/couchbase/sync_gateway/pull/2721

**Comment by adamcfraser** [comment]
Drone failure is unrelated intermittent test failure.

**Pull request merged**